lib/DDH/UserPage/Gather.pm

```diff
@@ -167,13 +179,17 @@ sub transform {
                         ( my $login = $developer->{url} ) =~
                             s{https://github.com/(.*)/?}{$1};
 
-                        my %contributor = (
-                            username => $login,
-                            avatar_url => $self->get_avatar($login)
-                        );
+                        $self->add_to_contributors($login, $ia, $ia_id, @contributors);
+                    } elsif ( $developer->{type} && 
+                              $developer->{type} eq 'duck.co' ) {
+
+                            my $complat_user = $self->ddgc->rs('User')->find({ username => $developer->{name} });
+
+                            if ( $complat_user && ( my $gh_id = $complat_user->github_id ) ) {
+                                my $gh_user = $self->ddgc->rs('GitHub::User')->find({ github_id => $gh_id });
```

Comment:
Do we want to check the `github_user_plaintext` column for people who haven't yet logged in with GH?

Down side of this is that data is completely unverified, but we generally act on good faith with people who have active contributions, so may not be a biggie?